packages/office-addin-dev-settings/src/commands.ts

```diff
@@ -233,6 +233,19 @@ export async function liveReload(manifestPath: string, command: commander.Comman
   }
 }
 
+function parseAppType(appType: string | undefined): AppType | undefined {
```

Comment:
Don't we already have this function somewhere? Let's try to use one copy.

Reply:
If we use the one in office-addin-debugging, I think I will need to move it to office-addin-dev-settings to avoid a circular dependency.  That's fine with me as long as we are in agreement to move it to office-addin-dev-settings

Reply:
I think it makes sense to do that. Thanks.

Reply:
Moving parseAppType to sideload.ts in office-addin-dev-settings.  I think it makes sense to put it in sideload.ts since this is where we also define the AppType enum